src/main/java/ca/landonjw/remoraids/implementation/listeners/forge/BossUpdateListener.java

```diff
@@ -30,59 +30,25 @@
  */
 public class BossUpdateListener {
 
-	/**
-	 * Invoked when a living entity updates.
-	 *
-	 * @param event event caused by living entities updating
-	 */
-	@SubscribeEvent
-	public void onUpdate(LivingEvent.LivingUpdateEvent event) {
```

Comment:
This event is called by every single entity to be checked to see if it's a raid boss. This was causing some minor performance issues, seen in timings as this is potentially called hundreds of thousands of times depending on how many entities your server has.

I spoke to Landon about it, and this exists so collisions can be cancelled as Reforged does not cancel collisions with statues. With this removed, I haven't seen any performance issues or lag spikes with 100+ players online and about 30-40 doing the raid boss.

For the boss size update, apparently Reforged did not serialize sizes when it was writing statues to the map, this is apparently fixed now so it's unnecessary.